cloud-init/setup_freeipa.sh

```diff
@@ -49,6 +49,13 @@ ipa-server-install --realm="${realm}" \
                    --admin-password="${admin_pw}" \
                    --hostname="${hostname}" \
                    --ip-address="$ip_address" \
+                   --http-cert-file=/etc/ipa/cert.p12 \
+                   --http-pin="zooble" \
```

Comment:
Possibly comment on `zooble`.

Reply:
See coommit 6fc041b.

Reply:
In case you were wondering, I took "zooble" from [a show my (much younger) brothers used to watch](https://en.wikipedia.org/wiki/Zoobilee_Zoo).

![image](https://user-images.githubusercontent.com/5521725/64747715-cfc27780-d4dd-11e9-8eed-9affb44817af.png)